pkg/assembler/backends/ent/backend/certify.go

```diff
@@ -53,20 +53,21 @@ func (b *EntBackend) CertifyGood(ctx context.Context, filter *model.CertifyGoodS
 	return collect(records, toModelCertifyGood), nil
 }
 
-func (b *EntBackend) IngestCertifyBad(ctx context.Context, subject model.PackageSourceOrArtifactInput, pkgMatchType *model.MatchFlags, spec model.CertifyBadInputSpec) (*model.CertifyBad, error) {
+func (b *EntBackend) IngestCertifyBadID(ctx context.Context, subject model.PackageSourceOrArtifactInput, pkgMatchType *model.MatchFlags, spec model.CertifyBadInputSpec) (string, error) {
```

Comment:
Should the return type be ID instead of string? Slightly concerned we might be using stringly-typed APIs.

Reply:
So it is specified as `ID`: https://github.com/guacsec/guac/blob/1116-return-ID/pkg/assembler/graphql/schema/certifyBad.graphql#L140 on the graphQL schema but translates to string in the resolvers.

Reply:
That makes sense, thanks for confirming!